app/assets/javascripts/client/controllers/assessments/AssessmentSharedDashboardSidebarCtrl.js

```diff
@@ -0,0 +1,25 @@
+PDRClient.controller('AssessmentSharedDashboardSidebarCtrl', [
```

Comment:
This particular controller is missing some key functionality which controls the display of the dates.